akka-stream/src/main/mima-filters/2.5.10.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.extrapolate")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.extrapolate$default$2")
```

Comment:
I'm assuming adding these ignores is acceptable, given the subject matter of the PR?

Reply:
Right, absolutely :)